### PR TITLE
os/binfmt : fix coding rules

### PR DESCRIPTION
**Pull request by manoharahk**
Signed-off-by: Manohara HK <manohara.hk@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [80bdd7f58b695c6d38a0ffe7278fb684f5f01cfd] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [80bdd7f58b695c6d38a0ffe7278fb684f5f01cfd] - Code Rule Check OK.

File: os/include/elf32.h [hunk 1]
```diff
@@ -74,41 +74,41 @@
 
 /* Values for Elf32_Ehdr::e_type */
 
-#define ET_NONE           0       /* No file type */
```
Comment (sunghan-chang) [inline os/include/elf32.h]: Before is not a violation. After can be. Because there are spaces before tab. Let's revert this change.

Comment (manoharahk) [reply os/include/elf32.h]: This change was done by cformarter.sh .. 
Is it wrong?

Comment (sunghan-chang) [reply os/include/elf32.h]: As far as I know, if there is a space before a tab. CRC recognizes it a violation. But I can be incorrect. And I actually prefer spaces than tabs for alignment.
